habitat_baselines/common/rollout_storage.py

```diff
@@ -161,6 +177,9 @@ def after_update(self):
             0 for _ in self.current_rollout_step_idxs
         ]
 
+        self.buffers["not_done_mask_0"] = torch.ones_like(self.buffers["not_done_mask_0"])
+        self.buffers["not_done_mask_1"] = torch.ones_like(self.buffers["not_done_mask_1"])
+
```

Comment:
```suggestion
```

Don't reset the masks. You could reset on the slice `[2:]` but you need to carry information over as otherwise `[0]` will never get set.

Reply:
I'll also put this here as I can't make a suggestion. But after
```
self.buffers[0] = self.buffers[self.current_rollout_step_idx]
```
You also need to do
```
self.buffers['not_done_mask_1'][1] = self.buffers['not_done_mask_1'][self.current_rollout_step_idx + 1]
```